window.go

```diff
@@ -246,6 +248,30 @@ func newWindow(ctx context.Context, l astikit.SeverityLogger, o Options, p Paths
 		return
 	})
 
+	// Bounds change handling, updates the internal WindowOption's bounds
+	updateBoundsFunc := func(w *Window) func(e Event) (deleteListener bool) {
+		return func(e Event) (deleteListener bool) {
+			w.m.Lock()
+			defer w.m.Unlock()
+			if w.o != nil && e.Bounds != nil {
+				w.o.X = e.Bounds.X
+				w.o.Y = e.Bounds.Y
+				w.o.Width = e.Bounds.Width
+				w.o.Height = e.Bounds.Height
+			}
+			return
+		}
+	}
+
+	w.On(EventNameWindowEventDidFinishLoad, updateBoundsFunc(w))
+	w.On(EventNameWindowEventMaximize, updateBoundsFunc(w))
+	w.On(EventNameWindowEventMove, updateBoundsFunc(w))
+	w.On(EventNameWindowEventMoved, updateBoundsFunc(w))
+	w.On(EventNameWindowEventResize, updateBoundsFunc(w))
+	w.On(EventNameWindowEventResizeContent, updateBoundsFunc(w))
+	w.On(EventNameWindowEventUnmaximize, updateBoundsFunc(w))
+	w.On(EventNameWindowEventWillMove, updateBoundsFunc(w))
+
```

Comment:
Can you also remove the portions of code that was updating window bounds before receiving a terminated event?

- https://github.com/asticode/go-astilectron/blob/master/window.go#L397
- https://github.com/asticode/go-astilectron/blob/master/window.go#L481
- https://github.com/asticode/go-astilectron/blob/master/window.go#L494
- https://github.com/asticode/go-astilectron/blob/master/window.go#L507

Reply:
Done